src/packages/solid/components/Metadata/Metadata.styles.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2023 Comcast Cable Communications Management, LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import theme from 'theme';
+
+const styles = {
+  container: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'flexStart'
+  },
+  titleTextStyle: {
+    ...theme.typography.headline3,
+    contain: 'both'
+  },
+  descriptionTextStyle: {
```

Comment:
i think it's a little redundant to call these `styles.titleTextStyle`, we can probably omit style and just call them `styles.titleText`